<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>746</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>HDB Loan Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(214, 214, 214);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1121</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="OverviewTab">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QTableWidget" name="OverviewTable">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>901</width>
        <height>501</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>991</width>
        <height>16777215</height>
       </size>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Invoice ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Company's Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Invoice Amount ($)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Factor Percentage (%)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Invoice Date</string>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credit Terms</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Factor Start Date</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>5</pointsize>
        </font>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="DeleteButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>530</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EditButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>530</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="InvoiceForm">
     <attribute name="title">
      <string>Invoice Form</string>
     </attribute>
     <widget class="QLabel" name="InvoiceFactoringHeading">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>-10</y>
        <width>710</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Invoice Factoring</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="NewInvoiceFrame">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>821</width>
        <height>421</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250,250,250);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="NewInvoiceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>New Invoice</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="InvoiceDateLineEdit">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>260</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>YYYY-MM-DD</string>
       </property>
      </widget>
      <widget class="QLabel" name="CompanyNameLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Company's Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="InvoiceAmtLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Invoice Amount ($)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="CreditTermsLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>320</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Credit Terms (Days)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="CreditTermsLineEdit">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>320</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>30</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="CompanyNameLineEdit">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>80</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>ABC Pte Ltd</string>
       </property>
      </widget>
      <widget class="QLabel" name="FactorPercentageLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Factor Percentage (%)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="FactorStartDayLineEdit">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>380</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>YYYY-MM-DD</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="InvoiceAmountLineEdit">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>140</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>50,000.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="InvoiceDateLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>260</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Invoice Date</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="FactorPercentageLineEdit">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>200</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>80</string>
       </property>
      </widget>
      <widget class="QLabel" name="FactoStartDayLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>380</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Factor Start Day (Optional)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="SaveButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>530</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ResetButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>530</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>1101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250,250,250);</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
